__tests__/selectors/benefits_test.js

```diff
@@ -18,84 +22,32 @@ describe("Benefits Selectors", () => {
     };
     state = {
       questions: questionsFixture,
-      benefits: [
```

Comment:
wow, I didn't know we were doing that...

Reply:
:(